packages/preview-middleware-client/src/cpe/changes/flex-change.ts

```diff
@@ -2,15 +2,16 @@ import type FlexCommand from 'sap/ui/rta/command/FlexCommand';
 import CommandFactory from 'sap/ui/rta/command/CommandFactory';
 import type { PropertyChange } from '@sap-ux-private/control-property-editor-common';
 import type { UI5AdaptationOptions } from '../types';
+import { validateBindingModel } from './validator';
 
 /**
  * Function to check a give value is a binding expression.
  *
- * @param value value to be checked.
+ * @param change changed property of a control
  * @returns boolean
  */
-function isBindingExpression(value: string): boolean {
-    return value.includes('{') && value.includes('}');
+function isBindingExpression(change: PropertyChange): boolean {
+    return typeof change.value === 'string' && change.value.includes('{') && change.value.includes('}');
```

Comment:
Maybe we can extract change.value here and add optional chaining (maybe the value can be null/undefined | change.value is duplicated in three places). 

Also we can include a check that '{' occurs before '}' like this `value.indexOf('{') < value.indexOf('}')`

Reply:
Done